guides/common/modules/con_mirroring-policies-overview.adoc

```diff
@@ -21,6 +21,9 @@ Complete Mirroring::
 Mirrors content as well as repodata.
 This is the fastest method.
 This mirroring policy is only available for Yum content.
-Republishing metadata in repositories with Complete Mirroring is high-risk.
-Hammer allows it using the `--force flag` command.
-This also applies to Content Views containing repositories with the Complete Mirroring policy.
++
+[WARNING]
+====
+{Team} does not recommend republishing metadata for repositories with Complete Mirror mirroring policy.
```

Comment:
Per the Red Hat style guide, I'd suggest avoiding the verb "recommend", especially in connection with {Team}: https://redhat-documentation.github.io/supplementary-style-guide/#_r

Possible alternatives include "Avoid republishing" or "Do not republish".